core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowApplication.scala

```diff
@@ -200,28 +261,29 @@ object CloudflowApplication {
 
           val st = containerStatus
             .flatMap(_.state match {
-              case Some(Container.Waiting(Some(reason))) if reason == "CrashLoopBackOff" ⇒ Some("CrashLoopBackOff")
+              case Some(Container.Waiting(Some(reason))) if reason == CrashLoopBackOff ⇒ Some(CrashLoopBackOff)
               case _ ⇒
                 status.phase.map {
-                  case Pod.Phase.Pending   ⇒ "ContainerCreating"
-                  case Pod.Phase.Running   ⇒ "Running"
-                  case Pod.Phase.Succeeded ⇒ "Terminated"
-                  case Pod.Phase.Failed    ⇒ "Error"
-                  case Pod.Phase.Unknown   ⇒ "Unknown"
+                  case Pod.Phase.Pending   ⇒ ContainerCreating
+                  case Pod.Phase.Running   ⇒ Running
+                  case Pod.Phase.Succeeded ⇒ Terminated
+                  case Pod.Phase.Failed    ⇒ Error
+                  case Pod.Phase.Unknown   ⇒ Unknown
                 }
             })
-            .getOrElse("Unknown")
+            .getOrElse(Unknown)
 
           PodStatus(name, st, restarts, ready)
         }
-        .getOrElse(PodStatus(name, "Unknown", 0, ""))
+        .getOrElse(PodStatus(name, Unknown, 0, ""))
     }
   }
 
   /**
    * Status of the pod.
    * ready can be "True", "False" or "Unknown"
    */
-  case class PodStatus(name: String, status: String, restarts: Int, ready: String)
-
+  case class PodStatus(name: String, status: String, restarts: Int, ready: String) {
```

Comment:
Can't we use `ready` from `object PodStatus` without making it a `String` ?

Reply:
It must be deprecated.

Reply:
(or removed, but older kubectl cloudflow will need it)